src/tactics/FStar.Tactics.Hooks.fs

```diff
@@ -380,6 +380,65 @@ let solve_implicits (env:Env.env) (tau:term) (imps:Env.implicits) : unit =
     end
   )
 
+(* Retrieves a tactic associated to a given attribute, if any *)
+let find_user_tac_for_attr env (a:term) : option<sigelt> =
+  let hooks = Env.lookup_attr env PC.resolve_implicits_attr_string in
```

Comment:
Oh this is related to my question about `resolve_implicits` attr in the example ... could you please say why are we requiring the `resolve_implicits` attribute also on the tactic?

Reply:
It's true that `resolve_implicits` is a bit misleading here. I wanted to convey the idea of `catch with tactic` that was already present with `resolve_implicits`, but I could indeed just look up for the `handle_smt_goals` attribute

Reply:
This is now fixed.